cime/scripts/tests/scripts_regression_tests.py

```diff
@@ -152,6 +152,17 @@ def test_lib_doctests(self):
                         run_cmd_assert_result(self, 'PYTHONPATH={}:$PYTHONPATH python -m doctest {} 2>&1'.format(LIB_DIR, filepath), from_dir=LIB_DIR)
                     else:
                         print("{} has no doctests".format(filepath))
+###############################################################################
```

Comment:
Nit: please put a newline between the end of a function and new class declaration.